tests/unit/info.tcl

```diff
@@ -410,7 +410,8 @@ start_server {tags {"info" "external:skip"}} {
             set info [r info clients]
             assert_equal [getInfoProperty $info pubsub_clients] {1}
             # non-pubsub clients should not be involved
-            assert_equal {0} [unsubscribe $rd2 {non-exist-chan}]
+            catch {unsubscribe $rd2 {non-exist-chan}} e
+            assert_match {*Can't execute 'unsubscribe' command in non-subscribe mode*} $e
```

Comment:
I prefer checking the uppercase error code like ERR or NOSUB in the test rather than the error message body.